docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst

```diff
@@ -0,0 +1,369 @@
+
+.. tabs::
+
+    .. tab:: Discovery *server*
+
+        .. code-block:: bash
+
+            # run the server docker image with the XML configuration
+            docker run -td --name discovery_server --net host eprosima/vulcanexus:humble
+            docker cp <workspace>/server_configuration.xml discovery_server:/server_configuration.xml
+            docker exec -it discovery_server bash
+
+            # run the discovery server
+            source /opt/vulcanexus/humble/setup.bash
+            fastdds discovery -i 0 -x server_configuration.xml
+
+    .. tab:: Listener
+
+        .. code-block:: bash
+
+            # run the listener docker image with the XML configuration
+            docker run -td --name ros_listener --net host eprosima/vulcanexus:humble
+            docker cp <workspace>/listener_configuration.xml ros_listener:/listener_configuration.xml
+            docker exec -it ros_listener bash
+
+            # run the listener client
+            source /opt/ros/humble/setup.bash
+            export FASTRTPS_DEFAULT_PROFILES_FILE="listener_configuration.xml"
+            export ROS_DISCOVERY_SERVER="TCPv4:[<server public IP address>]:10111"
+            ros2 run demo_nodes_cpp listener
+
+    .. tab:: Talker
+
+        .. code-block:: bash
+
+            # run the talker docker image with the XML configuration
+            docker run -td --name ros_talker --net host eprosima/vulcanexus:humble
+            docker cp <workspace>/talker_configuration.xml ros_talker:/talker_configuration.xml
+            docker exec -it ros_talker bash
+
+            # run the talker client
+            source /opt/ros/humble/setup.bash
+            export FASTRTPS_DEFAULT_PROFILES_FILE="talker_configuration.xml"
+            export ROS_DISCOVERY_SERVER="TCPv4:[<server public IP address>]:10111"
+            ros2 run demo_nodes_cpp talker
+
+Make sure that the discovery *server* public IP has been properly set in all sections of the XML configuration files (in both ``listener_configuration.xml``, ``server_configuration.xml`` and ``talker_configuration.xml``), and also in the environment variable ``ROS_DISCOVERY_SERVER``.
+The host public IP address can be obtained by running:
+
+.. code-block:: bash
+
+    wget -qO- http://ipecho.net/plain | xargs echo
+
+It is also required to set the WAN address for each *client* in the XML transport descriptor configuration.
+If any of them are hosted in the same LAN as the discovery *server*, then the IP address would be the same for both contexts (*client* and *server*).
+
+.. note::
+
+    If that is the case, then also make sure that the XML transport descriptor listening ports configured for each of them are different.
```

Comment:
NIT I think it is better to repeat "If both *client* and *server* share the same LAN,...." although perhaps it is a little bit weird to refer to a case that could not be previously read (because of being a note).

Reply:
I can move it to normal body text and remove the `note` tag.